spec/lib/api/2.0/hooks-spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Hooks v2.0', function () {
+    var hookService, Errors;
+    var hookSample = [{
+        url: "http://x.x.x.x",
+        name: "test",
+        filters: []
+    }];
+    var sandbox = sinon.sandbox.create();
+    before('start HTTP server', function () {
+        this.timeout(10000);
+        return helper.startServer([])
+        .then(function() {
+            hookService = helper.injector.get('Http.Services.Api.Hooks');
+            Errors = helper.injector.get('Errors');
+        });
+    });
+
+    afterEach('tear down mocks', function () {
+        sandbox.restore();
+    });
+
+    after('stop HTTP server', function () {
+        return helper.stopServer();
+    });
+
+    it('should return hooks', function () {
+        sandbox.stub(hookService, 'getHooks').resolves(hookSample);
+        return helper.request()
+            .get('/api/2.0/hooks')
+            .expect('Content-Type', /^application\/json/)
+            .expect(200, hookSample)
+            .expect(function (res) {
+                expect(hookService.getHooks).to.have.been.calledOnce;
+                expect(hookService.getHooks).to.have.been.calledWith({});
+            });
+    });
+
+    it('should post new hooks', function () {
+        sandbox.stub(hookService, 'createHook').resolves(hookSample[0]);
```

Comment:
In the viewer ( hook.2.0.json ), the `id` is required, but the stubbed return value doesn't contain a `id`, I think the unit-test will fail.